board/safety.h

```diff
@@ -504,16 +505,42 @@ bool steer_torque_cmd_checks(int desired_torque, int steer_req, const SteeringLi
     violation = true;
   }
 
-  // no torque if request bit isn't high
-  if ((steer_req == 0) && (desired_torque != 0)) {
-    violation = true;
+  bool steer_req_mismatch = (steer_req == 0) && (desired_torque != 0);
+
+  if (steer_req_mismatch) {
+    // no torque if request bit isn't high
+    if (!limits.has_steer_req_tolerance) {
+      violation = true;
+
+    // certain safety modes set their steer request bit low for one frame at a
+    // predefined max frequency to avoid steering faults in certain situations
```

Comment:
i'd move this comment up a bit at the top of this "section", think it's a bit easier to read that way